### PR TITLE
Add accept only number and fix dropdown input issue

### PR DESCRIPTION
**Pull request by kutsaniuk**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the Phone Input component to only accept numeric inputs, improving usability and data accuracy.

### CODE REVIEW

File: packages/@orchidui-vue/src/composables/helpers.js [hunk 1]
```diff
@@ -32,3 +32,29 @@ export const setCookie = (name, value, hours = 9600) => {
...
+  const { isAmountInput } = options;
+
+  const acceptedCharacters = [];
+
+  if (isAmountInput) {
+    acceptedCharacters.push(["."]);
+  }
+
+  if (
+    (event.key >= "0" && event.key <= "9") ||
```
Comment (kewcoder) [inline packages/@orchidui-vue/src/composables/helpers.js:46]: We have an issue when entering input using different keyboards, do we need to filter the input from the user, or just display an error message? cc @spacemonkey92 @mykhailo-diakovych

File: packages/@orchidui-vue/src/composables/helpers.js [hunk 2]
```diff
@@ -32,3 +32,29 @@ export const setCookie = (name, value, hours = 9600) => {
...
+    event.key === "ArrowLeft" ||
+    event.key === "ArrowRight" ||
+    event.key === "Tab" ||
+    event.key === "Backspace" ||
+    (event.key.startsWith("F") &&
```
Comment (kewcoder) [inline packages/@orchidui-vue/src/composables/helpers.js:51]: this will make users unable to copy/paste input

**Review by kewcoder — CHANGES_REQUESTED** [review]

**Comment by mykhailo-diakovych** [comment]
If we need only to write a number we could create filter on update:modalValue using this regex replace:

`  emit('update:model-value', value.replace(/[^0-9]/g, ''));
`
or use input type "number"
WDYT @kewcoder @kutsaniuk ?

**Comment by kutsaniuk** [comment]
> If we need only to write a number we could create filter on update:modalValue using this regex replace:
> 
> ` emit('update:model-value', value.replace(/[^0-9]/g, ''));`
> 
> or use input type "number"
> 
> WDYT @kewcoder @kutsaniuk ?

@mykhailo-diakovych it's change only value, but I need accept input only numbers. It's regarding this task https://linear.app/hitpay/issue/HIT-5662/add-edit-customer-in-order-details-page

File: packages/@orchidui-vue/src/composables/helpers.js (hunk 2 above)
Comment (kewcoder) [reply packages/@orchidui-vue/src/composables/helpers.js:51]: @kutsaniuk copy/paste function make sure it works

File: packages/@orchidui-vue/src/Form/PhoneInput/OcPhoneInput.vue [hunk 3]
```diff
@@ -87,6 +89,33 @@ const scrollToSelectedCountry = () => {
...
 };
+
+const onPaste = (e) => {
+  let text = e.clipboardData.getData("Text");
+
+  if (text.replace(/[^0-9]/g, "")) {
+    text = text.slice(0, 19);
+
+    if (text.length > 5) {
+      const { nationalNumber, countryCallingCode, country } = parsePhoneNumber(
```
Comment (kewcoder) [inline packages/@orchidui-vue/src/Form/PhoneInput/OcPhoneInput.vue:100]: add try catch here

**Review by kewcoder — APPROVED** [review]
good job

**Comment by kutsaniuk** [comment]
@mykhailo-diakovych I used library for parsing number onPaste

**Review by mykhailo-diakovych — APPROVED** [review]

**Pull request merged**